File: 성규/8주차/구명보트/PGS_Lv2_42885.js
function solution(people, limit) {
  let count = 0;
  people.sort((a, b) => a - b);

  // 가장 무거운 사람 , 가장 가벼운 사람이 함께 탈 수 있는지 검사
  for (let i = 0, j = people.length - 1; i <= j; j--) {
    if (people[i] + people[j] <= limit) {
      count++;
      i++;
    } else {
      count++; // 아니면 무거운사람 먼저 보트 탑승
    }
  }
  return count;
}

File: 수빈/src/PGS_Lv2_42885.java
import java.util.Arrays;

public class PGS_Lv2_42885 {
    public int solution(int[] people, int limit) {
        int answer = 0;
        Arrays.sort(people);

        int index = 0;
        for (int i = people.length - 1; i >= index; i--) {
            if (people[index] + people[i] <= limit) {
                index++;
            }
            answer++;
        }

        return answer;
    }
}